### DISCUSSION TITLE
brew wouldn’t install

### DISCUSSION THREAD

**Discussion by s0m-a** [Homebrew/discussions#5017] [Getting started]
### Output of `brew config`

```shell
zsh: command not found: brew
```

### Output of `brew doctor`

```shell
zsh: command not found: brew
```

### Description of issue

soma@somas-MBP ~ % /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"

==> Checking for `sudo` access (which may request your password)...
Password:
==> This script will install:
/usr/local/bin/brew
/usr/local/share/doc/homebrew
/usr/local/share/man/man1/brew.1
/usr/local/share/zsh/site-functions/_brew
/usr/local/etc/bash_completion.d/brew
/usr/local/Homebrew

Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R soma:admin /usr/local/Homebrew
==> Downloading and installing Homebrew...
remote: Enumerating objects: 252246, done.
remote: Counting objects: 100% (40843/40843), done.
remote: Compressing objects: 100% (858/858), done.
fatal: the remote end hung up unexpectedly.74 MiB | 96.00 KiB/s 
fatal: early EOFs:  11% (28945/252246), 13.89 MiB | 105.00 KiB/s
fatal: index-pack failed
Failed during: /usr/bin/git fetch --force origin
soma@somas-MBP ~ %

**Comment by SMillerDev** [comment]
See #666
